package.json

```diff
@@ -136,11 +138,13 @@
     "prom-client": "^12.0.0",
     "proxy-agent": "^3.1.1",
     "querystringify": "^2.1.1",
+    "react-use": "^15.3.4",
     "source-map-support": "^0.5.16",
     "stack-utils": "^2.0.1",
     "striptags": "^3.1.1",
     "tsscmp": "^1.0.6",
     "url-regex": "^5.0.0",
+    "use-resize-observer": "^6.1.0",
```

Comment:
We actually already have our own resize observer provided by:

```ts
import { useResizeObserver } from "coral-framework/hooks";
```

Reply:
If this provides a better API though, I'd replace our implementation with the package one.

Reply:
It does provide a better API in my opinion, I've replaced our implementation in the one place it was used.